evm/src/main/java/org/hyperledger/besu/evm/gascalculator/ConstantinopleGasCalculator.java

```diff
@@ -39,12 +37,24 @@ public class ConstantinopleGasCalculator extends ByzantiumGasCalculator {
 
   private static final long EXTCODE_HASH_COST = 400L;
 
+  /**
+   * Returns the amount of gas the CREATE2 operation will consume.
+   *
+   * @param frame The current frame
+   * @return the amount of gas the CREATE2 operation will consume
+   * @deprecated Compose the operation cost from {@link #txCreateCost()}, {@link
```

Comment:
Perhaps a mention of the Create2Opcode.cost() method since it is implemented there?

Reply:
Also added to javadoc in for createOperationGasCost.